owtf/managers/config.py

```diff
@@ -32,7 +32,7 @@ def load_config_file(file_path, fallback_file_path):
         # check if the config file exists
         abort_framework("Config file not found at: {}".format(file_path))
     try:
-        config_map = yaml.load(FileOperations.open(file_path, "r"))
+        config_map = yaml.load(FileOperations.open(file_path, "r"), Loader=yaml.FullLoader)
```

Comment:
this is actually very insecure - can you use `safe_load` function provided by the PyYAML? it would be great if you  can also fix this anywhere we load YAML files 🙀

Reply:
Sure will do that.